[DataViews]: bump to 0.1.1

retrofox opened the pull request:
<!--
Link a related issue to this PR. If the PR does not immediately resolve the issue,
for example, it requires a separate deployment to production, avoid
using the "fixes" keyword and instead attach the [Status] Fix Inbound label to
the linked issue.
-->

Related to ARC-161

## Proposed Changes

* Bumped the version of `@automattic/dataviews` to `0.1.1`.

## Why are these changes being made?
<!--
It's easy to see what a PR does but much harder to find out why it was made,
particularly when researching old changes in history. Record an explanation of
the motivation behind this change and how it will help.
-->

* This version bump reflects recent internal changes aimed at improving flexibility in UI composition using DataViews.

## Testing Instructions

<!--
Add as many details as possible to help others reproduce the issue and test the fix.
"Before / After" screenshots can also be very helpful when the change is visual.
-->

* Confirm the new version is correctly published in `package.json`.

## Pre-merge Checklist

<!--
Complete applicable items on this checklist **before** merging into trunk. Inapplicable items can be left unchecked.

Both the PR author and reviewer are responsible for ensuring the checklist is completed.
-->

- [ ] Has the general commit checklist been followed? (PCYsg-hS-p2)
- [ ] [Have you written new tests](https://wpcalypso.wordpress.com/devdocs/docs/testing/index.md) for your changes?
- [ ] Have you tested the feature in Simple (P9HQHe-k8-p2), Atomic (P9HQHe-jW-p2), and self-hosted Jetpack sites (PCYsg-g6b-p2)?
- [ ] Have you checked for TypeScript, React or other console errors?
- [ ] Have you used memoizing on expensive computations? More info in [Memoizing with create-selector](https://github.com/Automattic/wp-calypso/blob/trunk/packages/state-utils/src/create-selector/README.md) and [Using memoizing selectors](https://react-redux.js.org/api/hooks#using-memoizing-selectors) and [Our Approach to Data](https://github.com/Automattic/wp-calypso/blob/trunk/docs/our-approach-to-data.md)
- [ ] Have we added the "[Status] String Freeze" label as soon as any new strings were ready for translation (p4TIVU-5Jq-p2)?
    - [ ] For UI changes, have we tested the change in various languages (for example, ES, PT, FR, or DE)? The length of text and words vary significantly between languages. 
- [ ] For changes affecting Jetpack: Have we added the "[Status] Needs Privacy Updates" label if this pull request changes what data or activity we track or use (p4TIVU-aUh-p2)?

retrofox committed:
bump @automattic/dataviews@0.1.1

oandregal commented on packages/dataviews/CHANGELOG.automattic.md:
```diff
@@ -1,6 +1,8 @@
-## Next 
+## Next
+
+## 0.1.1
```
Can I suggest changes to the changelog before publishing the package?

Instead of:

```md
- Extract `<DefaultUI />` as a standalone component to clarify layout logic and prepare for custom UI compositions.
- Expose `<DataViews.BulkActionToolbar />` component.
- Expose `<DataViews.Filters />` component.
- Expose `<DataViews.FiltersToggle />` component.
- Expose `<DataViews.Layout />` component.
- Expose `<DataViews.LayoutSwitcher />` component.
- Expose `<DataViews.Pagination />` component.
- Expose `<DataViews.Search />` component.
- Expose `<DataViews.ViewConfig />` component.
- Add support for `free-composition` in the `DataViews` component
- Fix `filterSortAndPaginate` to handle undefined values for the `is` filter.
```

I'd do:

```md
- Add support for free composition in the `DataViews` component by exporting subcomponents: `<DataViews.ViewConfig />`, `<DataViews.Search />`, `<DataViews.Pagination />`, `<DataViews.LayoutSwitcher />`, `<DataViews.Layout />`, `<DataViews.FiltersToggle />`, `<DataViews.Filters />`, `<DataViews.BulkActionToolbar />`.
- Fix `filterSortAndPaginate` to handle undefined values for the `is` filter.
```

- Removes any reference to `DefaultUI`: this is something internal that consumers won't understand.
- Renames `free-composition` to free composition. The former looks like it's a new "mode" you activate or prop, but it isn't.
- Group everything together.

retrofox replied:
> * Removes any reference to `DefaultUI`: this is something internal that consumers won't understand.

Yes, I was unsure if we should add it to the changelog. Removing it makes sense.

retrofox replied:
Addressed 1290217dd6a4643bc01c67d2280d39c8e39a2370

retrofox committed:
tidy changelog
@see https://github.com/Automattic/wp-calypso/pull/103164#pullrequestreview-2818104539

oandregal reviewed (approved):
Thanks, I suppose you're interested in a new version to use in woo analytics, right?

For the record, there's a field guide with steps to publish the npm package at PCYsg-173c-p2
